### ISSUE TITLE
train  cls always nan...

### ISSUE THREAD

**Issue by AiIsBetter** [DetectionTeamUCAS/FCOS_Tensorflow#1]
I modified your program ,train according to my own data set, and in the process of training, CLS loss is always nan,

 2019-04-27 22:02:27: step90    image_name:b'70844.jpg' |	 loc_loss:11.512925148010254 |	 
                                            cls_loss:nan |	 cnt_loss:0.6479541063308716 |	total_loss:nan |
                                            per_cost_time:8.343699216842651s

I downloaded the pre-training model and put it in the designated place
cfgs.py:CLASS_NUM = 11
labe_dict:
elif  cfgs.DATASET_NAME == 'xxxx':
    NAME_LABEL_MAP = {
        '0': 0,
        '1': 1,
        '2': 2,
        '3': 3,
        '4': 4,
        '5': 5,
        '6': 6,
        '7': 7,
        '8': 8,
        '9': 9,
        '10': 10,
        '11': 11
    }
'0'  is  background
Is there anything else I need to change?

**Comment by yangxue0827** [comment]
Sorry, I am still debugging this code, and I have only experimented on the coco and voc2007 datasets. Maybe you can try to adjust the learning rate and warm up （use multi_gpu_train.py）. @AiIsBetter

**Comment by AiIsBetter** [comment]
> Sorry, I am still debugging this code, and I have only experimented on the coco and voc2007 datasets. Maybe you can try to adjust the learning rate and warm up （use multi_gpu_train.py）. @AiIsBetter

Yes, it's normal to change the learning rate. I forgot about it. But as you say, the results are worse. Hope to use the debugging code as soon as possible. Thank you。 But your code is enough for me to study the paper, haha.

**Comment by LCWdmlearning** [comment]
> > Sorry, I am still debugging this code, and I have only experimented on the coco and voc2007 datasets. Maybe you can try to adjust the learning rate and warm up （use multi_gpu_train.py）. @AiIsBetter
> 
> Yes, it's normal to change the learning rate. I forgot about it. But as you say, the results are worse. Hope to use the debugging code as soon as possible. Thank you。 But your code is enough for me to study the paper, haha.

hello ,have you solved this problem yet?，I have the same problem.
